articles/container-instances/container-instances-tutorial-deploy-app.md

```diff
@@ -47,7 +47,11 @@ Container registry password:
 az acr credential show --name <acrName> --query "passwords[0].value"
 ```
 
-To deploy your container image from the container registry with a resource request of 1 CPU core and 1 GB of memory, run the following command. Replace `<acrLoginServer>` and `<acrPassword>` with the values you obtained from the previous two commands.
+To deploy your container image from the container registry with a
+resource request of 1 CPU core and 1 GB of memory, run the following
+command, [`az container create`][az-container-create]. Replace `<acrLoginServer>` and `<acrPassword>` with the
+values you obtained from the previous two commands and
+`aci-tutorial-app` with the name you want to give the new container. 
```

Comment:
It's not strictly required to specify a different name for the container, so I'd recommend dropping that part of the sentence to keep things as concise as possible. However, if you feel strongly about it and wish to keep it, add a comma after "commands".

Reply:
Not really, you are right. But it's also a good idea to substitute "commands" by "markers". I'll propose a new version.